docs/sources/configure-client/aws-lambda.md

```diff
@@ -0,0 +1,99 @@
+Since Lambda functions are billed based on execution time and allocated memory, code inefficiencies can lead to higher costs. Often, these costs accumulate unnoticed because of the following reasons:
+
+* **Granular billing**: Lambda functions are billed in milliseconds, which can make small inefficiencies seem insignificant at first. However, when scaled to thousands or millions of invocations, these inefficiencies can lead to substantial costs.
+
+* **Complex performance profile**: Lambda functions may interact with various services and resources, making it challenging to pinpoint performance bottlenecks.
+
+* **Variable load**: The serverless nature of AWS Lambda means that functions might handle variable loads at different times, making it hard to optimize for every scenario.
+
+Profiling Lambda functions helps identify these hidden performance bottlenecks, enabling developers to optimize their code for both performance and cost.
+Effective profiling can reveal inefficient code paths, unnecessary memory usage, and areas where the execution time can be reduced.
+By addressing these issues, organizations can significantly reduce their AWS bill, improve application responsiveness, and ensure a more efficient use of resources.
+
+## Architecture
+
+This extension runs a relay server on the same network namespace as the Lambda function, ensuring minimal added latency.
+
+![Lambda Extension Architecture](https://user-images.githubusercontent.com/23323466/186037668-44de7caa-6576-422a-b3f7-8416325f4a98.png)
+
+For more details, refer to the [Building Extensions for AWS Lambda blog post](https://aws.amazon.com/blogs/compute/building-extensions-for-aws-lambda-in-preview/).
+
+## Set up the Pyroscope Lambda extension
+
+To set up the Pyroscope Lamnda extension, you need to: 
+
+1. Configure your Lamda function
+1. Set up your environment variables 
+1. Integrate the Pyroscope SDK
+
+### Configure your Lambda function
+
+Configure your Lambda function to use the extension. Find the latest release on our [releases page](https://github.com/grafana/pyroscope-lambda-extension/releases).
+
+### Set up the environment variables 
+
+Configure the extension with the following environment variables:
+
+| Environment Variable           | Default Value                           | Description                                  |
+| ------------------------------ | --------------------------------------- | -------------------------------------------- |
+| `PYROSCOPE_REMOTE_ADDRESS`     | `https://profiles-prod-001.grafana.net` | Destination for relayed Pyroscope data       |
+| `PYROSCOPE_BASIC_AUTH_USER`    | `""`                                    | HTTP Basic authentication user               |
+| `PYROSCOPE_BASIC_AUTH_PASSWORD`| `""`                                    | HTTP Basic authentication password           |
+| `PYROSCOPE_AUTH_TOKEN`         | `""`                                    | Authorization key (token authentication)     |
```

Comment:
There are 4 from 9 options that control authentication and authorisation:
 - `PYROSCOPE_BASIC_AUTH_USER` and `PYROSCOPE_BASIC_AUTH_PASSWORD` required for cloud.
 - `PYROSCOPE_TENANT_ID` might be needed for on-premises deployment.
 - `PYROSCOPE_AUTH_TOKEN` was required for OG cloud, and might still be useful if pyroscope is deployed on-premises and has some custom authentication layer that uses bearer token.

I think we could remove the latter from this page to avoid overloading users with rarely useful information.

```suggestion
```